### PR TITLE
CorrDiff: Change dir based on job name

### PR DESCRIPTION
**Pull request by mnabian**
# Modulus Pull Request

## Checklist

- [x] I am familiar with the [Contributing Guidelines](https://github.com/NVIDIA/modulus/blob/main/CONTRIBUTING.md).
- [x] New or existing tests cover these changes.
- [x] The documentation is up to date with these changes.
- [ ] The [CHANGELOG.md](https://github.com/NVIDIA/modulus/blob/main/CHANGELOG.md) is up to date with these changes.
- [ ] An [issue](https://github.com/NVIDIA/modulus/issues) is linked to this pull request.

### CODE REVIEW

**Review by MortezaMardani — APPROVED** [review]

File: examples/generative/corrdiff/train.py [hunk 1]
```diff
@@ -141,10 +141,10 @@ def main(cfg: DictConfig) -> None:
...
 
     # Save a copy of the Modulus source code
-    if dist.rank == 0:
-        shutil.copytree(
-            os.path.dirname(modulus.__file__), "modulus", dirs_exist_ok=True
-        )
+    # if dist.rank == 0:
+    #    shutil.copytree(
+    #        os.path.dirname(modulus.__file__), "modulus", dirs_exist_ok=True
+    #    )
```
[the remark this answers is not in the document]
Comment (mnabian) [reply examples/generative/corrdiff/train.py:146]: Let me keep this here for now as I need to fix it in a separate PR

**Pull request merged**